set.mm

```diff
@@ -266723,6 +267201,96 @@ left ideal which is also a right ideal (or a left ideal over the opposite
       STUTAKUIVHUTCUJCTOUTTOCDUKVHCVGTFAPUIULUTCUMUNUOUQUR $.
   $}
 
+  ${
+    $d B x y $.  $d I x y $.  $d R x y $.  $d U x y $.
+    isridl.u $e |- U = ( LIdeal ` ( oppR ` R ) ) $.
+    isridl.b $e |- B = ( Base ` R ) $.
+    isridl.t $e |- .x. = ( .r ` R ) $.
+    $( A right ideal is a left ideal of the opposite ring.  This theorem shows
+       that this definition corresponds to the usual textbook definition of a
+       right ideal of a ring to be a subgroup of the additive group of the ring
+       which is closed under right-multiplication by elements of the full ring.
+       (Contributed by AV, 13-Feb-2025.) $)
+    isridl $p |- ( R e. Ring -> ( I e. U <-> ( I e. ( SubGrp ` R )
+                               /\ A. x e. B A. y e. I ( y .x. x ) e. I ) ) ) $=
+      ( crg wcel cfv csubg cv co wral wa wb eqid coppr opprring opprbas dflidl2
+      cmulr syl wceq opprsubg eqcomi a1i eleq2d opprmul eleq1i ralbidva anbi12d
+      bitrd ) DKLZGFLZGDUAMZNMZLZAOZBOZUSUEMZPZGLZBGQZACQZRZGDNMZLZVCVBEPZGLZBG
+      QZACQZRUQUSKLURVISDUSUSTZUBABCUSVDFGHCDUSVPIUCVDTZUDUFUQVAVKVHVOUQUTVJGUT
+      VJUGUQVJUTDUSVPUHUIUJUKUQVGVNACUQVBCLRZVFVMBGVFVMSVRVCGLRVEVLGCDVDEUSVBVC
+      IJVPVQULUMUJUNUNUOUP $.
```

Comment:
Possibly name this theorem `dfridl2`?
I understand there is no `~ df-ridl`, though...

Reply:
> Possibly name this theorem `dfridl2`? I understand there is no `~ df-ridl`, though...

Yes, since there is no definition 1, then there can't be a definition 2.

Reply:
Yes, fine!